ziti/src/main/kotlin/org/openziti/net/ZitiServerSocketChannel.kt

```diff
@@ -70,7 +70,7 @@ internal class ZitiServerSocketChannel(val ctx: ZitiContextImpl): AsynchronousSe
             State.closed -> throw ClosedChannelException()
         }
 
-        val servResult = runCatching { ctx.getService(local.service) }
+        val servResult = runCatching { ctx.getService(local.service, 5000L) }
```

Comment:
i assume this times out at 5s and before it never timed out? that may not be long enough for crappy networks, why not something much larger like 30/60?